zebra-chain/src/block/commitment.rs

```diff
@@ -159,6 +160,62 @@ impl From<ChainHistoryMmrRootHash> for [u8; 32] {
     }
 }
 
+impl ChainHistoryMmrRootHash {
```

Comment:
Should the new hex formatting code be behind a feature flag?

(Some of these types already had partial hex support, so it's a bit complicated to isolate it.)

Reply:
I think this is okay without a feature flag.